internal/eval/sortedset/sorted_set.go

```diff
@@ -251,3 +251,32 @@ func (ss *Set) CountInRange(minVal, maxVal float64) int {
 
 	return count
 }
+
+// GetScoreRange returns a slice of members with scores between min and max, inclusive.
+// If withScores is true, the members will be returned with their scores.
+func (ss *Set) GetMemberScoresInRange(minScore, maxScore float64, count, max int) ([]string, []float64) {
+	var members []string
+	var scores []float64
+
+	iterFunc := func(item btree.Item) bool {
+		ssi := item.(*Item)
+		if ssi.Score < minScore {
```

Comment:
yinqiwen uses min score inclusive and max score exclusive.

Reply:
This code does as well, right? 

We return true without appending to members for everything below minScore. This keeps the iterator iterating without appending, but starts appending once >= minScore. 
We return false for everything equal or greater to maxScore to stop iterating and appending for good.

Either way, it seems it's not clear, so I can add a test case for this.

Reply:
Yeah. Sorry for the confusion. 
+1